tornado-api/src/main/java/uk/ac/manchester/tornado/api/profiler/ChromeEventTracer.java

```diff
@@ -0,0 +1,109 @@
+package uk.ac.manchester.tornado.api.profiler;
+
+import java.io.*;
+import java.util.Map;
+
+public class ChromeEventTracer {
```

Comment:
Is possible this class implements the profiler interface?

Reply:
No.  Started with that, but many of the API's don't make sense.